tools/telemetry-generator/src/handlers/appInsightsExecutionTimeTestHandler.ts

```diff
@@ -15,11 +15,11 @@ module.exports = function handler(fileData, telemetryClient: TelemetryClient): v
 		const arithmeticMeanMetricName = `${fileData.suiteName}_${testData.benchmarkName}_arithmeticMean`;
 		try {
 			console.log(
-				`emitting metric ${arithmeticMeanMetricName} with value ${testData.stats.arithmeticMean}`,
+				`emitting metric ${arithmeticMeanMetricName} with value ${testData.customData["Period (ns/op)"]}`,
```

Comment:
Since we already had a problem with strictly abiding by the format of the key (e.g., capitalization, spacing), how do you think about assigning constant variables to each string keys? 

Similar example would be how we created variables for `devtools`:
[ConnectContainer](https://github.com/microsoft/FluidFramework/blob/f85a3d2e1e4f599194cb85c62ab24f029dcd34b7/packages/tools/devtools/devtools-core/src/messaging/container-devtools-messages/ConnectContainer.ts#L20)
[Usage](https://github.com/microsoft/FluidFramework/blob/f85a3d2e1e4f599194cb85c62ab24f029dcd34b7/packages/tools/devtools/devtools-core/src/ContainerDevtools.ts#L268)

I don't we have to create an object like the `devtools` example.

Reply:
I'm not too sure I like the approach in this case. For execution time and memory benchmarks, we technically can because benchmark tool is the "owner" of the custom data, so it knows exactly which keys will appear. But for custom benchmarks, the tool will just receive keys that the test authors come up with, so fundamentally we can't provide constants for those. Having different "behaviors" depending on the kind of benchmark is not the end of the world, but it feels a bit weird to me.

Also, let's think about who would use those constants. Right now we only use them in the handlers in telemetry-generator, which does not have a dependency on benchmark-tool. If we create them, that means telemetry-generator will want a new dependency on benchmark-tool so it can use them. Again, not the end of the world, but I'm still unsure what I think about that dependency existing in the first place. I think I lean towards being ok with it, but since it requires another release of benchmark tool, I'd suggest we capture an item in the backlog but skip it for now.